go/pod/pod.go

```diff
@@ -34,13 +43,22 @@ func validatePrivateKeyHex(pk string) error {
 	return nil
 }
 
+var downloadOnce sync.Once
+var downloadErr error
+var podWorkerPath string // Final path to the extracted pod_worker binary
+
 func dispatchRustCommand(req podCommandRequest) (*Pod, string, error) {
+	binPath, err := getOrDownloadPodWorker()
```

Comment:
I would only get here, and fail if its not there. I think we should force people to explicitly Init and wait for the file to be downloaded before booting up their server, vs having the first request potentially be slow or fail given it will download in case it's not present.